03/app.js

```diff
@@ -1,3 +1,12 @@
-const x = 10;
-let iteration = 0;
-let randomNumber = -1;
\ No newline at end of file
+let iteration = 1;
+const x = 5;
+const randomNumber = Math.round(Math.random() * 20);
+
+while (x !== randomNumber) {
+  const randomNumber = Math.round(Math.random() * 20);
+  if (x === randomNumber) {
```

Comment:
Ten warunek nie jest potrzebny bo ten problem rozwiązuje już warunek w pętli `while` jeśli liczba jest znaleziona to po prostu pętla przestaje działać.